src/Initialization/init_Symbolics.jl

```diff
@@ -1,50 +1,51 @@
 eval(quote
     using .Symbolics: gradient,
-                            simplify,
-                            Num,  # variable like, e.g. x[1]
-                            Term, # term like, eg. x[1] + x[2] == 1
-                            Symbolic,
-                            operation,
-                            arguments
+                      simplify,
+                      Num,  # variable like, e.g. x[1]
+                      Term, # term like, eg. x[1] + x[2] == 1
+                      Symbolic,
+                      operation,
+                      arguments
 
-   """
-       _vec(vars::NTuple{L, Union{<:Num, <:Vector{Num}}}) where {L}
+    """
+        _vec(vars)
 
-   Transform a tuple of operations into one vector of operations.
+    Transform a tuple of operations into one vector of operations.
 
-   ### Input
+    ### Input
 
-   - `vars` -- tuple where each element is either variable-like (`Num`) or a
-               vector of variables (`Vector{Num}`)
+    - `vars` -- tuple where each element is either variable-like (`Num`) or a
+                vector of variables (`Vector{Num}`)
 
-   ### Output
+    ### Output
 
-   A vector of `Operation` obtained by concatenating each tuple component.
+    A vector of `Operation` obtained by concatenating each tuple component.
 
-   ## Examples
+    ## Examples
 
-   ```julia
-   julia> vars = @variables x[1:2] y
-   (Num[x₁, x₂], y)
+    ```jldoctest
+    julia> using Symbolics
 
-   julia> LazySets._vec(vars)
-   3-element Vector{Num}:
-    x₁
-    x₂
-    y
-   ```
-   """
-    function _vec(vars::NTuple{L, Union{<:Num, <:Vector{Num}}}) where {L}
-        return collect(reduce(vcat, vars))
-    end
+    julia> vars = @variables x[1:2] y
+    2-element Vector{Any}:
+      x[1:2]
+     y
 
-    # case with a single variable
-    _vec(vars::Tuple{Num}) = [vars[1]]
+    julia> LazySets._vec(vars)
+    3-element Vector{Num}:
+     x[1]
+     x[2]
+        y
+    ```
+    """
+    function _vec end
 
     # reduce for several variables e.g. when vars = @variables x[1:3] t
     _vec(vars::Vector{Any}) = reduce(vcat, vars)
     _vec(vars::Vector{Num}) = vars
+    _vec(vars::Vector{Symbolics.Arr{Num, 1}}) = reduce(vcat, vars)
     _vec(vars::Vector{Vector{Num}}) = reduce(vcat, vars)
+    _vec(vars::Vector{Real}) = reduce(vcat, vars)
```

Comment:
since only the implementation for `::Vector{Num}` differs from the rest, i wonder if we should simplify this by doing:

```julia
_vec(vars::Vector) = reduce(vcat, vars) # fallback
_vec(vars::Vector{Num}) = vars # no-op
```

Reply:
I do not understand the logic here, so I cannot say if that is correct. The catch-all will apply to other cases and that may be wrong. If you think it is correct, we can change it.

Reply:
I guess so, but we can also simplify that part in the future.